src/plugins/auto/executable_network.cpp

```diff
@@ -655,7 +655,7 @@ InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetMetric(const std::st
                 real = _loadContext[CPU].
                     executableNetwork->GetMetric(name).as<unsigned int>();
             }
-            unsigned int res = std::max(8u, real);
+            unsigned int res = (std::max)(8u, real);
```

Comment:
which header includes Windows.h? We need to remove Windows.h from headers and use only from cpp

Reply:
Hi Ilya, the including is in plugins/auto/utils/thread_utils.hpp line 20.

Reply:
@wangleis, @tiger100256-hu fyi

Reply:
@WenjingKangIntel , thank you very much, I will create a PR to remove Windows.h in thread_utils.hpp.

Reply:
@WenjingKangIntel Thanks for your input of this issue. Since issue will be fixed in another way, this PR will be closed.